.env.test

```diff
@@ -0,0 +1,21 @@
+# Sets the location of the flask application object: https://flask.palletsprojects.com/en/2.3.x/cli/#application-discovery
+# Needed because of the presence of the "todo_app" folder
+FLASK_APP=todo_app/app
+
+# Turn on debug mode (which enables reloading on code changes and the interactive debugger: https://flask.palletsprojects.com/en/2.3.x/config/#DEBUG)
+FLASK_DEBUG=true
+
+# Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
+SECRET_KEY=secret-key
+
+TRELLO_API_KEY = 'TRELLO_API_KEY'
```

Comment:
Really minor but it might catch you out later; I'd advise sticking to the format of these strings above like `TRELLO_API_KEY=TRELLO_API_KEY` rather than with quotes and spaces; debatable whether it's as readable but unfortunately whilst the Python-dotenv library is happy to interpret sensibly, when we come to Docker it will pass this through literally, and it can be really confusing when you're debugging and the string `'TRELLO_API_KEY'` is being printed and it's hard to spot how that's different to `TRELLO_API_KEY` without the quotes!